### DISCUSSION TITLE
Seeking Recommendations for an Easy to Use Custom Design Tool for My Website

### DISCUSSION THREAD

**Discussion by PattiRubio** [community/community#146374] [Programming Help]
### Select Topic Area

Question

### Body

Hi everyone,

I run a website focused on custom packaging solutions (you can check it out here: [Wax Paperie](https://customwaxpaper.com/)) and I've been brainstorming ways to enhance the user experience for my customers.

One feature I'm keen to implement is a custom art design option where visitors can create their own artwork directly on the website. The idea is for them to easily design and submit their creations when requesting a quote. However, I’m looking for a tool that is:

User-friendly for customers with varying design skills.
Easy to integrate with an existing website.
Supports file export in formats like PNG or PDF for print-ready artwork.

I’ve looked into a few options like Zakeke, Lumise and Canva API but I’d love to hear your recommendations or experiences with similar tools.

If you’ve implemented something like this or have ideas about the best approach, I’d greatly appreciate your insights.

Thanks in advance for your guidance! 😊

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @PattiRubio!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
